example/Components/AdvancedDeepNestedData.js

```diff
@@ -13,45 +13,90 @@ const {
 
 const Form = ({ revalidation: { form, onChange, errors = {}, onSubmit }, onSubmit: submitCb }) =>
   (
-    <div className='form'>
+    <form className='form' noValidate onSubmit={(e) => { e.preventDefault(); onSubmit(({valid, form}) => valid ? submitCb(form) : null ) }}>
```

Comment:
using a real `form` element helps screen readers enter "forms mode". The `novalidate` attribute suppresses the native error message "bubbles".